sn_node/src/quote.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use sn_protocol::error::Error as ProtocolError;
+use sn_protocol::NetworkAddress;
+use sn_transfers::{NanoTokens, PaymentQuote};
+
+use crate::node::Node;
+
+/// The time in seconds that a quote is valid for
+const QUOTE_EXPIRATION_SECS: u64 = 3600;
+
+impl Node {
+    pub(crate) fn create_quote_for_storecost(
```

Comment:
lets get some more docs/info around these. How the quotes are envisaged to be used etc. (here and below)